app/modules/ia_config_reader.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import json
+import os.path as path
+
+log = logging.getLogger(__name__)
+
+
+def load_config_to_dict(fname):
+    app_home = ''
+    config_path = path.join(app_home, 'ia-configs', fname)
+    assert path.isfile(config_path), f'Could not find config at path {config_path}'
+    with open(config_path, 'r') as file:
+        config_dict = json.load(file)
+    return config_dict
+
+
+def short_config_name_to_full_filename(config_name):
```

Comment:
I'm really not seeing the benefit inm aking this a function. It's only called once in the real code and the call could be replaced by this one line. 
The tests could be done by calling the IaConfig constructor